libcsm.spec

```diff
@@ -70,7 +70,7 @@ Cray System Management procedures and operations.
 %{buildroot}%{install_python_dir}/bin/python -m pip uninstall -y pip setuptools wheel
 
 # Fix the virtualenv activation script, ensure VIRTUAL_ENV points to the installed location on the system.
-sed -i -E 's:^(VIRTUAL_ENV=).*:\1'%{install_python_dir}':' %{buildroot}%{install_python_dir}/bin/activate
+find %{buildroot}%{install_python_dir}/bin -type f | xargs -t -i sed -i 's:%{buildroot}%{install_python_dir}:%{install_python_dir}:g' {}
```

Comment:
Can skip the xargs and just find ... -exec ... {} \; but I like saving fork()/exec()'s.

Reply:
yeah, it was being funny and I was pretty impatient with it so I copied the xargs bit from docs-csm for ipmitool and called it a day.